example/src/main/java/com/stripe/example/activity/SimpleRedirectPaymentMethodActivity.kt

```diff
@@ -0,0 +1,205 @@
+        super.onDestroy()
+    }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        stripe.onPaymentResult(requestCode, data, PaymentIntentResultCallback(this))
+    }
+
+    private fun createPaymentIntent(country: String, params: PaymentMethodCreateParams) {
+        compositeSubscription.add(
+            backendApi
+                .createPaymentIntent(
+                    mutableMapOf(
+                        "amount" to 1000,
+                        "country" to country
+                    )
+                )
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .doOnSubscribe {
+                    disableUi()
+                    viewBinding.status.setText(R.string.creating_payment_intent)
+                }
+                .subscribe(
+                    { handleCreatePaymentIntentResponse(it, params) },
+                    { handleError(it) }
+                )
+        )
+    }
+
+    private fun disableUi() {
+        viewBinding.submit.isEnabled = false
+        viewBinding.name.isEnabled = false
+        viewBinding.email.isEnabled = false
+        viewBinding.progressBar.visibility = View.VISIBLE
+    }
+
+    private fun handleError(err: Throwable) {
+        viewBinding.submit.isEnabled = true
+        viewBinding.name.isEnabled = true
+        viewBinding.email.isEnabled = true
+        viewBinding.progressBar.visibility = View.INVISIBLE
+        viewBinding.status.append("\n\n" + err.message)
+    }
+
+    private fun handleCreatePaymentIntentResponse(
+        responseBody: ResponseBody,
+        params: PaymentMethodCreateParams
+    ) {
+        try {
+            val responseData = JSONObject(responseBody.string())
+            viewBinding.status.append("\n\n" + getString(R.string.payment_intent_status,
+                responseData.getString("status")))
+            val secret = responseData.getString("secret")
+            confirmPaymentIntent(secret, params)
+        } catch (e: IOException) {
+            handleError(e)
+        } catch (e: JSONException) {
+            handleError(e)
+        }
+    }
+
+    private fun confirmPaymentIntent(
+        paymentIntentClientSecret: String,
+        params: PaymentMethodCreateParams
+    ) {
+        viewBinding.status.append("\n\nStarting payment authentication")
+        stripe.confirmPayment(
+            this,
+            ConfirmPaymentIntentParams.createWithPaymentMethodCreateParams(
+                paymentMethodCreateParams = params,
+                clientSecret = paymentIntentClientSecret,
+                returnUrl = "example://return_url"
+            )
+        )
+    }
+
+    private fun onConfirmSuccess(result: PaymentIntentResult) {
+        val paymentIntent = result.intent
+        viewBinding.status.append("\n\n" +
+            "Auth outcome: ${result.outcome}\n\n" +
```

Comment:
"PaymentIntent confirmation outcome: ..."

You can also change this to a multiline string literal